content/blog/2018/03/2018-03-19-introducing-jenkins-x.adoc

```diff
@@ -0,0 +1,203 @@
+---
+layout: post
+title: "Introducing Jenkins X: a CI/CD solution for modern cloud applications on Kubernetes"
+tags:
+- jenkins-x
+- kubernetes
+- pipeline
+author: jstrachan
+---
+
+We are excited to share and invite the community to join us on a project we’ve been thinking about over the last few months called http://jenkins-x.io[Jenkins X] which extends the Jenkins ecosystem to solve the problem of automating CI / CD in the cloud.
+
+== Background
+
+The last few years have seen massive changes in the software industry:
+
+* use of immutable container images for distributing software which are smaller, easier to work with and have cheaper hardware costs than VMs (approx 20% less on average)
+* https://kubernetes.io/[Kubernetes] has become the defacto way of installing, upgrading, operating and managing containers at scale on any public or hybrid cloud 
+** 2018 is the year all the major public clouds, operating system vendors and PaaS offerings support https://kubernetes.io/[Kubernetes] natively
+** we now have open source industry standard for distributing, installing and managing applications on any cloud!
+* increased adoption of microservices and cloud native applications leading to massive increase in the number of components which require CI / CD along with increased release frequency
+* improvements in DevOps practices coming from the community such as the https://puppet.com/blog/2017-state-devops-report-here[State of DevOps Report] which show the approach of high performing teams
+** increasingly many businesses now realise that to compete you have to deliver value quickly via software
+** teams need to become high performing if the business is to succeed
+
+All of this adds up to an increased demand for teams to have a solution for cloud native CI / CD with lots of automation!
+
+== Introducing Jenkins X
+
+http://jenkins-x.io[Jenkins X] is a project which rethinks how developers should interact with CI / CD in the cloud with a focus on making development teams productive through automation, tooling and DevOps best practices.
+
+http://jenkins-x.io[Jenkins X] is open source and we invite you to give us feedback and to contribute to the project.
+
+image::http://jenkins-x.io/images/logo.png[Sunset,200,200]
+
+Here's a brief overview of the current Jenkins X features and towards the end of this post there's a short video demo.
```

Comment:
@jstrachan 
What do you think of the idea of moving the TL;DR up here?  The list of features is interesting but I think it will be more engaging to talk about the the overall vision (which is in the TL;DR) first. 

I think it would also work better to have the video immediately following the the introduction and the list of features following that.  

What do you think?

Reply:
great point - lemme do that